Sources/SwiftDocC/Utility/MarkupExtensions/ListItemExtractor.swift

```diff
@@ -59,8 +59,16 @@ extension Sequence where Element == InlineMarkup {
         guard let colonIndex = initialText.firstIndex(of: ":") else {
             return nil
         }
-
-        let nameStartIndex = initialText[...colonIndex].lastIndex(of: " ").map { initialText.index(after: $0) } ?? initialText.startIndex
+        
+        let nameStartIndex: String.Index
+        if skipStartingTag == true, var spaceIndex = initialText[...colonIndex].firstIndex(of: " ") {
+            while initialText[spaceIndex] == " " {
+                spaceIndex = initialText.index(spaceIndex, offsetBy: 1)
+            }
```

Comment:
Directly manipulating a string index and using it to access the content in a loop like this is a bad way of working with Strings in Swift. It's possibly accidentally quadratic because the characters are not known to be the same number of bytes.

I think this is trying to do something like the below to find the start of the next if there are multiple spaces between the colon and the word
```swift
let nameStartIndex = initialText[...colonIndex].drop(while: { $0 != " " }).dropFirst().firstIndex(where: { $0 != " " }) 
    ?? initialText.startIndex
```

in which case it might be more robust to use `$0.isWhitespace` to also handle tabs and other whitespace characters.

```swift
let nameStartIndex = initialText[...colonIndex].drop(while: { !$0.isWhitespace }).dropFirst().firstIndex(where: { !$0.isWhitespace }) 
    ?? initialText.startIndex
```

Regardless. It would be good to add a short comment to describe why this is happening.